pkg/consumer/consumer.go

```diff
@@ -4,4 +4,5 @@ import "io"
 
 type Consumer interface {
 	Consume(reader io.Reader, destPath string) error
+	SetOverwriteTarget(force bool)
```

Comment:
eeh, I don't like this interface. We should pass a slice/varargs of ConsumeOptions (of which OverwriteTarget would be one instance) into Consume() instead.